packages/zend-file/library/Zend/File/ClassFileLocator.php

```diff
@@ -68,6 +68,12 @@ public function __construct($dirOrIterator = '.')
                 define('T_NS_SEPARATOR', '\\');
             }
         }
+
+        if (PHP_VERSION_ID < 80000) {
+            if (!defined('T_NAME_QUALIFIED')) {
+                define('T_NAME_QUALIFIED', '');
```

Comment:
```suggestion
                define('T_NAME_QUALIFIED', 0);
```

those defines are numeric, so let's keep the type?

Reply:
Doesn't really matter because the sole purpose of the define is to not have undefined constants, but sure why not.